common/url_utils.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/strings/utf_string_conversions.h"
+#include "base/strings/string_util.h"
+#include "brave/common/url_constants.h"
+#include "chrome/common/webui_url_constants.h"
+
+namespace brave {
+  base::string16 ReplaceChromeSchemeWithBrave(const base::string16 url) {
```

Comment:
I think this can actually be removed if we fix the URL fixup code to swap out chrome:// with brave://. I'm looking into that at the moment.